### ISSUE TITLE
p1 and p2 offscreen

### ISSUE THREAD

**Issue by snoyface** [optexture/paint#4]
during testing with shui, p1 and p2 were not on screen, even with dot size at maximum. Also p3 was not seen, but we were only testing x axis.

**Comment by t3kt** [comment]
What x values were coming in for p1 and p2? We may need to adjust the rendering scale.

**Comment by JimAnkrom** [comment]
We were sending (and seemed to be receiving) data per spec tho... to be
clear.

**Comment by JimAnkrom** [comment]
For x... I think we were sending 0-360. Ok, so maybe that wasn't spec.

**Comment by t3kt** [comment]
The spec says 0-1.
We can either scale it in sway or in the renderer. It doesn't really matter which.

**Comment by t3kt** [comment]
If we do decide to do it in the renderer, we should also update the test osc generator to match it.

**Comment by JimAnkrom** [comment]
No, no, the client software does all translations. The problem part is
converting the compass heading to be 0-1 within the installation space's
x-axis.  .

No worries, will sort that out..
